### DISCUSSION TITLE
please help to find a version

### DISCUSSION THREAD

**Discussion by rodkevich** [Kong/insomnia#8264] [Help] [answered]
Could you please let me know the latest stable version of the application for macos that did not include ‘organizations’ and ‘request parameter locking’ and 'new ui panel' and 'welcome panel'?
I accidentally updated the version and discovered that the app became the same, which is why I stopped using Postman and switched to Insomnia

**Comment by subnetmarco** [comment, marked as answer]
Hey @rodkevich , you probably are looking at a version before 8.x. In what ways is Insomnia the same as Postman? What is "request parameter locking"?

You may want to give it a try anyways:

* Your data can be local only
* Unlimited collection runner
* Pre-request and after-response scripting
* New mocking capability
* New global environment variables

And a lot more. You could go back at using the older version, but you would really be missing out on all the new capabilities.

**Reply by rodkevich** [reply]
Hello
https://github.com/Kong/insomnia/issues/8265 - "request parameter locking"
Yes thank you very much for reply
It's true - new versions offer cool new features, for those who need them
But I'm a minimalist) I just need a couple of buttons and for them to work the way I like) well you know, 
without any authorizations, a bunch of panels and other things turning an application into an ide
And a wonderful system of plugins helps to add what I'm missing
